Source/Processors/Editors/ListSliceParser.cpp

```diff
@@ -0,0 +1,186 @@
+/*
+------------------------------------------------------------------
+
+This file is part of the Open Ephys GUI
+Copyright (C) 2013 Open Ephys
+
+------------------------------------------------------------------
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+#include "ListSliceParser.h"
+#include <vector>
+
+
+ListSliceParser::ListSliceParser(std::string s)
+{
+    defaultString = s;
+}
+
+ListSliceParser::~ListSliceParser()
+{
+
+}
+
+int ListSliceParser::convertToInteger(std::string s)
+{
+    char ar[20];
+    int i, j, k = 0;
```

Comment:
As I said before, method convertToInteger() is deprecated, since it's better to use something like [std::stoi](http://www.cplusplus.com/reference/string/stoi/) because we have C++11 support.

Or do you have any reasons to use your custom method?